datalad/resources/website/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href="http://datalad.org/pics/favicon-16x16.png" />
+	<link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
+          body{
+           max-width: 1200px;
+           margin: 2em auto;
+         }
+	 td.dir {font-weight: bold; color: #00A1CB; }
+	 td.git {font-weight: bold; color: #00BA88; }
+	 td.annex {font-weight: bold; color: #EC6C20; }
+	 td.link-broken {color:#E53B51; }
+	 td.link {color: #FFD03C;  }
+	 td.file {color: #67CDDC;  }
+	</style>
+    </head>
+    <body>
+        <h1><img src='http://datalad.org/pics/favicon.svg' width="24">Datalad Repository</h1>
+	<table id="directory" class="display" width="100%" cellspacing="0"></table>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.12.3.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/jquery.dataTables.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="https://raw.githubusercontent.com/blueimp/JavaScript-MD5/master/js/md5.min.js"></script>
```

Comment:
Yeah, but should I change them to pull from our github repo for now, i.e somethings like https://raw.githubusercontent.com/datalad/datalad/example.js ? It would be pretty slow that way actually and the way i'm pulling the md5.js script isn't ideal either.
I could put the css, js files under '.git/datalad' of the dataset too on `create-publication`
this is where the generated metadata is currently being stored, don't know if its a good idea though